src/sentry/mail/adapter.py

```diff
@@ -93,7 +94,7 @@ def get_sendable_user_objects(project):
         notifications for the provided project.
         """
         recipients_by_provider = NotificationSetting.objects.get_notification_recipients(project)
-        return {user for users in recipients_by_provider.values() for user in users}
+        return recipients_by_provider.get(ExternalProviders.EMAIL)
```

Comment:
Can you add a unit test for this static method?

Reply:
@AniketDas-Tekky Ok, just pushed a change to existing tests that breaks without this change